sandy/core/cov.py

```diff
@@ -621,6 +621,35 @@ def from_stdev(cls, std):
         var = std_ * std_
         return CategoryCov.from_var(var)
 
+    def _gls_Vy_calc(self, S):
+        """
+        2D calculated output using S.T.dot(Vx_prior).dot(S)

```

Comment:
Here the transpose is not in the same place than in the method

Reply:
I dont know what is the difference. In the probability statistics and data uncertainties in nuclear science and technology appears in this way, and the model is y_a=f(theta_a). In our case is y_a=Q*theta_a, so i copy because i dont really the difference

Reply:
I mean that in the docstring and in the function you don't do the same...probably you just need to fix the docstring